pages/linux/hdparm.md

```diff
@@ -0,0 +1,27 @@
+# hdparm
+
+> Get/set SATA/IDE device (HDD) parameters.
+
+- Request identification info of the device directly from the drive:
+
+`sudo hdparm -I /dev/{{device}}`
+
+- Get Advanced Power Management feature, if the drive supports it:
```

Comment:
Perhaps this will read better?
```suggestion
- Get the Advanced Power Management level:
```

(does this still mean the same thing?)

Reply:
yes, it's cleaner